### PR TITLE
Feature/svg spritemap with fragment identifiers

### PR DESCRIPTION
**Pull request by pjzmilestone**

### CODE REVIEW

File: src/scss/rich-icons.scss [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
...0 1.57.32 2.134.89a3.03 3.03 0 0 1 .879 2.145v20.048a3.493 3.493 0 0 0 3.475 3.489h36.787a3.468 3.468 0 0 0 2.464-1.018 3.496 3.496 0 0 0 1.022-2.471V41.154h-.473z' id='Path' fill-rule='nonzero'/%3e%3cpath id='Rectangle' fill='white' fill-rule='nonzero' d='M115.622 24.563h16.365v1h-16.365z'/%3e%3cpath class='animate-line-long' stroke='white' stroke-width='.5' fill='white' d='M164.47 118.149a3.106 3.106 0 0 1-.905 2.203 3.082 3.082 0 0 1-2.194.908h-20.734v-9.722h-.452v9.722H88.411a3.116 3.116 0 0 1-3.099-3.111v-6.211h-.452v6.21c0 1.971 1.588 3.57 3.55 3.576h72.982a3.533 3.533 0 0 0 2.518-1.044 3.561 3.561 0 0 0 1.033-2.531V55.985h-.452l-.021 62.164z' id='Path' fill-rule='nonzero'/%3e%3cpath class='animate-line' stroke='white' stroke-width='.5' fill='white' d='M77.694 114.012a2.08 2.08 0 0 1-2.076 2.074H18.485a2.05 2.05 0 0 1-1.471-.6c-.39-.39-.609-.921-.606-1.474v-5.023h-.452v5.023c-.006.674.26 1.322.735 1.797a2.491 2.491 0 0 0 1.794.73h57.133a2.513 2.513 0 0 0 1.787-.736 2.533 2.533 0 0 0 .741-1.791v-1.826h-.452v1.826z' id='Path' fill-rule='nonzero'/%3e%3cpath d='M119.947 104.776h-5.509V68.915c0-.613-.244-1.2-.678-1.631a2.32 2.32 0 0 0-1.625-.67H52.431a2.32 2.32 0 0 0-1.625.67 2.29
```
Comment (ms-Danny) [inline src/scss/rich-icons.scss]: this can be removed

File: src/svg/rich/cam-group-small.svg [hunk 2]
```diff
@@ -0,0 +1,731 @@
...
+}
+
+.scale-height {
+  animation-duration: 3000ms;
+  animation-iteration-count: infinite;
+  transform-origin: 50% 86%;
+  animation-name: scale-height;
+}
+
+/* .scale-height1 {
```
Comment (ms-Danny) [inline src/svg/rich/cam-group-small.svg]: Guess we need a task for cleaning up in the svg codes

Comment (pjzmilestone) [reply src/svg/rich/cam-group-small.svg]: We have svg optimizer which cleanup and minify code automatically.

File: src/scss/rich-icons.scss (hunk 1 above)
Comment (pjzmilestone) [reply src/scss/rich-icons.scss]: this file is in gitignore (I just removed it from git tracking)

**Pull request merged**